pkgs/tools/games/opentracker/default.nix

```diff
@@ -1,38 +1,43 @@
-{ lib
-, stdenv
-, buildDotnetModule
-, fetchFromGitHub
-, autoPatchelfHook
-, wrapGAppsHook
-, dotnetCorePackages
-, fontconfig
-, gtk3
-, openssl
-, libX11
-, libXi
-, xinput
+{
+  lib,
+  stdenv,
+  buildDotnetModule,
+  fetchFromGitHub,
+  autoPatchelfHook,
+  wrapGAppsHook,
+  dotnetCorePackages,
+  fontconfig,
+  gtk3,
+  libunwind,
+  openssl,
+  xinput,
+  xorg,
 }:
-
 buildDotnetModule rec {
   pname = "opentracker";
-  version = "1.8.2";
+  version = "1.8.5";
 
   src = fetchFromGitHub {
     owner = "trippsc2";
     repo = pname;
-    rev = version;
-    sha256 = "0nsmyb1wd86465iri9jxl3jp74gxkscvnmr3687ddbia3dv4fz0z";
+    rev = "refs/tags/${version}";
```

Comment:
This is needed? It doesnt work with just `version`?

Reply:
This is a little bit defensive coding, but I received it as a comment in another PR, so I've started making it in new PRs.

`version` is ambiguous: it could be a branch or a tag. `refs/tags/${version}` makes it clear, to the reader and to `git`, that it's a tag. The branch equivalent is `refs/heads/${version}`. See https://github.com/NixOS/nixpkgs/blob/690eb0d43e51ec138454c2d6b689d7b697a3399f/doc/builders/fetchers.chapter.md#fetchgit-fetchgit